osf/management/commands/data_storage_usage.py

```diff
@@ -492,10 +495,11 @@ def process_usages(
         ('registrations', 0),
         ('nd_quick_files', 0),
         ('nd_public_nodes', 0),
+        ('nd_private_nodes', 0),
```

Comment:
Good reminder about how once we're running this for real, to tack new columns on the end!

Reply:
For sure.